compute_sdk/globus_compute_sdk/errors/error_types.py

```diff
@@ -87,4 +108,8 @@ def __init__(
         self.completion_t = completion_t or str(time.time())
 
     def __str__(self) -> str:
-        return "\n" + textwrap.indent(self.remote_data, " ")
+        stack_trace = textwrap.indent(self.remote_data, " ")
+        message = "\n" + stack_trace
+        if re.search(TaskExecutionFailed.SERDE_REGEX, stack_trace):
+            message += SERDE_TASK_EXECUTION_FAILED_HELP_MESSAGE
+        return message
```

Comment:
I generally like this; any reason not to include a test for it?

Reply:
Test added